integration-libs/digital-payments/src/checkout/cms-components/dp-payment-method/dp-payment-callback/dp-payment-callback.component.ts

```diff
@@ -0,0 +1,72 @@
+import { DpLocalStorageService } from './../../../facade/dp-local-storage.service';
+import { DP_CARD_REGISTRATION_STATUS } from '../../../../utils/dp-constants';
+import { ActivatedRoute } from '@angular/router';
+import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
+import { DpCheckoutPaymentService } from '../../../facade';
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'cx-dp-payment-callback',
+  templateUrl: './dp-payment-callback.component.html',
+})
+export class DpPaymentCallbackComponent implements OnInit {
+  @Output()
+  closeCallback = new EventEmitter<any>();
+  @Output()
+  paymentDetailsAdded = new EventEmitter<any>();
+
+  constructor(
+    private dpPaymentService: DpCheckoutPaymentService,
+    private dpStorageService: DpLocalStorageService,
+    private globalMsgService: GlobalMessageService,
+    private route: ActivatedRoute
```

Comment:
For extensibility we use `protected` for contructor parameters

Reply:
done